addons/web_tour/static/src/js/tip.js

```diff
@@ -259,12 +259,14 @@ var Tip = Widget.extend({
         }
 
         if (this.viewPortState === 'in') {
-            this.$el.position({
-                my: this._get_spaced_inverted_position(appendAt),
-                at: appendAt,
-                of: this.$anchor,
-                collision: "none",
-            });
+            if(!this.tip_opened) {
+                this.$el.position({
+                    my: this._get_spaced_inverted_position(appendAt),
+                    at: appendAt,
+                    of: this.$anchor,
+                    collision: "none",
+                });
+            }
```

Comment:
In fact, is any code in the `reposition` function useful when the tip is opened? I think there was a check before that was removed, maybe this is the whole cause of the problem?

Reply:
can you please review it, as we are now allowing to reposition only if the tooltip info is not opened.